README.md

```diff
@@ -102,6 +103,7 @@ Cette table ````curiosities```` a plusieurs colonnes (block ````do````):
 
 Les deux dernières colonnes (created_at et updated_at) sont créées grace à la ligne ````t.timestamps null: false````, ajoutée automatiquement par Ruby On Rails. 
 
+Pour vous aider un peu, nous avons déjà lancé cette première migration "CreateCuriosities".
```

Comment:
As the migration has not been applied for help but to manage the exercice, I suggest something like:

Dans le cadre de l'exercice, notez que la migration ````CreateCuriosity``` a déjà été appliquée à la base de données.